labels.json

```diff
@@ -139,6 +139,16 @@
     "description": "Feature Request Pending SRS",
     "color": "f9d0c4"
   },
+  {
+    "name": "status: requires prd",
+    "description": "Feature Request Pending Product/Feature Requirement Doc",
+    "color": "159ce2"
+  },
+  {
+    "name": "status: requires planning",
```

Comment:
This will be used to hold new features (epics) over to larger team planning sessions before we progress and development. Status: blocked is not really suitable, this provides me better filtering for those sessions.